### PR TITLE
Updated UA dependency in podspec to v9.1, updated target to iOS 10

### PR DESCRIPTION
**Pull request by anuragbhatnagar**
Updated UA dependency in podspec to v9.1 from v8.2.2, updated deployment target to iOS 10

### CODE REVIEW

**Comment by anuragbhatnagar** [comment]
@rlepinski

File: Segment-UrbanAirship.podspec.json [hunk 1]
```diff
@@ -10,10 +10,10 @@
   },
   "source": {
     "git": "https://github.com/urbanairship/ios-segment-integration.git",
-    "tag": "1.1.0"
+    "tag": "1.2.0"
   },
   "platforms": {
-    "ios": "8.0"
+    "ios": "10.0"
```
Comment (rlepinski) [inline Segment-UrbanAirship.podspec.json]: This can be set to 9

**Comment by rlepinski** [comment]
Update target could be set to 9, we will continue to support it until iOS 12 comes out. Other than that it looks good.

**Comment by anuragbhatnagar** [comment]
@rlepinski Ready for merge

**Comment by marc-scig** [comment]
👍 thanks @anuragbhatnagar

**Pull request merged**